src/ios-deploy/ios-deploy.m

```diff
@@ -1836,6 +1881,7 @@ void usage(const char* app) {
         @"  -L, --justlaunch             just launch the app and exit lldb\n"
         @"  -v, --verbose                enable verbose output\n"
         @"  -m, --noinstall              directly start debugging without app install (-d not required)\n"
+        @"  -A, --app_deltas             incremental install. must specify a directory to store app deltas to determine what needs to be installed\n"
```

Comment:
This should appear in the same location that it does in the longopts[] array and the string passed to getopt_long().

Reply:
Done.